crates/proof/src/cert_validity.rs

```diff
@@ -0,0 +1,44 @@
+use alloy_primitives::B256;
+use eigenda_v2_struct_rust::EigenDAV2Cert;
+use risc0_zkvm::is_dev_mode;
+use risc0_zkvm::Receipt;
+
+#[derive(Debug, Clone, Default)]
+pub struct CertValidity {
+    /// the claim about if the cert is valid
+    pub claimed_validity: bool,
+    /// a zkvm proof attesting the above result
+    /// in dev mode, receipt is ignored
```

Comment:
```suggestion
    /// A zkvm proof attesting the claimed_validity.
    /// In dev mode, receipt is ignored.
```

Reply:
Best to use full punctuation, otherwise it reads like "proof attesting the above result in dev mode", aka that the proof is only used in dev mode.